controllers/handlers/kubevirt_test.go

```diff
@@ -1964,6 +1975,51 @@ Version: 1.2.3`)
 					Expect(existingResource.Annotations).ToNot(HaveKey(kubevirtcorev1.EmulatorThreadCompleteToEvenParity))
 				})
 
+				It("should add the Passt Binding to Kubevirt CR if PasstNetworkBinding is true in HyperConverged CR", func() {
+					hco.Annotations[deployPasstNetworkBindingAnnotation] = "true"
+					hco.Spec.NetworkBinding = nil
+
+					existingResource, err := NewKubeVirt(hco)
+					Expect(err).ToNot(HaveOccurred())
+
+					Expect(existingResource.Spec.Configuration.NetworkConfiguration).NotTo(BeNil())
+					Expect(existingResource.Spec.Configuration.NetworkConfiguration.Binding).NotTo(BeNil())
+					expectedInterfaceBindingPlugin := kubevirtcorev1.InterfaceBindingPlugin{
```

Comment:
maybe add an helper function to generate it? I think the test will be more readable.